transformer/redshift_types.go

```diff
@@ -68,9 +70,8 @@ func GetTransform(t_type string) ColumnTranformer {
 // New types should register here
 var (
 	transformMap = map[string]ColumnTranformer{
-		"int":          intFormat,
-		"bigint":       intFormat,
-		"int8":         intFormat,
+		"int":          intFormat(32),
+		"bigint":       intFormat(64),
```

Comment:
Is there a practical benefit to having both forms of int? Why not just make everything 64bit?